acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -32,25 +32,34 @@
       Convert.toWei(TRANSFER_AMOUNT_ETHER, Convert.Unit.ETHER).toBigIntegerExact();
 
   @Test
-  public void ethGetTransactionCountReturnCorrectNumber() {
-    final BigInteger startTransactionCount =
+  public void ethGetTransactionCountReturnCorrectNumber() throws Exception {
+    final BigInteger pantheonTransactionCount =
         ethNode().transactions().count(richBenefactor().address());
-    submitTransactionAndWaitForBlock();
+    BigInteger ethSignerTransactionCount =
+        ethSigner().transactions().count(richBenefactor().address());
 
-    final BigInteger endTransactionCount =
-        ethNode().transactions().count(richBenefactor().address());
+    assertThat(pantheonTransactionCount).isEqualTo(ethSignerTransactionCount);
+
+    submitTransactionAndWaitForBlock(); // should increase the transaction count by one.
+
+    ethSignerTransactionCount = ethSigner().transactions().count(richBenefactor().address());
 
-    assertThat(startTransactionCount.add(BigInteger.ONE)).isEqualTo(endTransactionCount);
+    assertThat(pantheonTransactionCount.add(BigInteger.ONE)).isEqualTo(ethSignerTransactionCount);
   }
 
   @Test
-  public void ethBalanceRequesReturnsCorrectBalance() {
-    final BigInteger startBalance = ethNode().accounts().balance(RECIPIENT);
+  public void ethBalanceRequesReturnsCorrectBalance() throws Exception {
+    final BigInteger startBalance = ethSigner().accounts().balance(RECIPIENT);
+
     submitTransactionAndWaitForBlock();
 
-    final BigInteger endBalance = ethNode().accounts().balance(RECIPIENT);
+    final BigInteger endBalance = ethSigner().accounts().balance(RECIPIENT);
 
     assertThat(endBalance).isEqualByComparingTo(startBalance.add(TRANSFER_AMOUNT_WEI));
+
+    final BigInteger pantheonBalance = ethNode().accounts().balance(RECIPIENT);
+
+    assertThat(pantheonBalance).isEqualTo(endBalance);
```

Comment:
perhaps prefix endBalance with ethSigner to make clear we are comparing values obtained directly and through the signer?

Reply:
done